cmd/template/docker/files/docker-compose/mongo.tmpl

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+
+services:
+  mongo:
+    image: mongo:latest
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: ${DB_USERNAME}
+      MONGO_INITDB_ROOT_PASSWORD: ${DB_ROOT_PASSWORD}
+    ports:
+      - "${DB_PORT}:27017"
```

Comment:
*Adding a general comment here that applies to all the DB templates*
The thing I am noticing is that if the user does not have their .env set up - obviously these will not work.

So what I am thinking is to do what create-t3-app does, which is basically spin up a .env.example file that has placeholders or somthing that then a user can use to remake into their own .env

That way they can set it up themselves and not have to worry about it not working.
What do you think?
Mayve in the .env.example we can also pre-fill these values to show a user what they will need.